web/styles/compose.css

```diff
@@ -802,6 +802,15 @@ input.recipient_box {
         padding: 0 5px;
         position: relative;
         top: 0.7px;
+
+        .drafts-count {
+            margin-left: 3px;
+
+            /* Dont show count on smaller widths due to space restrictions. */
+            @media (width < calc( $md_min + 32px)) {
```

Comment:
I'm not a fan of this `calc()` bit, because then the list of breakpoints no longer really describes where they layout might shift. If there's not enough space at `md_min`, let's either include another breakpoint or just accept things as they are for now and shift to a larger breakpoint here.